MapboxNavigation/PollyVoiceController.swift

```diff
@@ -1,162 +0,0 @@
-     */
-    public var globalVoiceId: AWSPollyVoiceId?
-    
-    /**
-     `regionType` specifies what AWS region to use for Polly.
-     */
-    public var regionType: AWSRegionType = .USEast1
-    
-    /**
-     `identityPoolId` is a required value for using AWS Polly voice instead of iOS's built in AVSpeechSynthesizer.
-     You can get a token here: http://docs.aws.amazon.com/mobile/sdkforios/developerguide/cognito-auth-aws-identity-for-ios.html
-     */
-    public var identityPoolId: String
-    
-    var pollyTask: URLSessionDataTask?
-    
-    public init(identityPoolId: String) {
-        self.identityPoolId = identityPoolId
-        
-        let credentialsProvider = AWSCognitoCredentialsProvider(regionType: regionType, identityPoolId: identityPoolId)
-        let configuration = AWSServiceConfiguration(region: regionType, credentialsProvider: credentialsProvider)
-        AWSServiceManager.default().defaultServiceConfiguration = configuration
-        
-        super.init()
-    }
-    
-    public override func alertLevelDidChange(notification: NSNotification) {
-        guard shouldSpeak(for: notification) == true else { return }
-        
-        let routeProgresss = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationRouteProgressKey] as! RouteProgress
-        let userDistances = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationDistanceToEndOfManeuverKey] as! CLLocationDistance
-        let instruction = spokenInstructionFormatter.string(routeProgress: routeProgresss, userDistance: userDistances, markUpWithSSML: true)
-        
-        speak(instruction, error: nil)
-        startAnnouncementTimer()
-    }
-    
-    override func speak(_ text: String, error: String?) {
-        assert(!text.isEmpty)
-        
-        let input = AWSPollySynthesizeSpeechURLBuilderRequest()
-        input.textType = .ssml
-        input.outputFormat = .mp3
-        
-        let langs = Locale.preferredLocalLanguageCountryCode.components(separatedBy: "-")
-        let langCode = langs[0]
-        var countryCode = ""
-        if langs.count > 1 {
-            countryCode = langs[1]
-        }
-        
-        switch (langCode, countryCode) {
-        case ("de", _):
-            input.voiceId = .marlene
-        case ("en", "CA"):
-            input.voiceId = .joanna
-        case ("en", "GB"):
-            input.voiceId = .brian
-        case ("en", "AU"):
-            input.voiceId = .nicole
-        case ("en", "IN"):
-            input.voiceId = .raveena
-        case ("en", _):
-            input.voiceId = .joanna
-        case ("es", _):
-            input.voiceId = .miguel
-        case ("fr", _):
-            input.voiceId = .celine
-        case ("it", _):
-            input.voiceId = .giorgio
-        case ("nl", _):
-            input.voiceId = .lotte
-        case ("ru", _):
-            input.voiceId = .maxim
-        case ("sv", _):
-            input.voiceId = .astrid
```

Comment:
Will the Voice API choose a voice based on the locale?

Reply:
Yeah, I'll add this to the list.